bq-connector/docai_bq_connector/doc_ai_processing/Processor.py

```diff
@@ -213,38 +224,53 @@ def _process_async(self) -> Union[DocumentOperation, ProcessedDocument]:
         storage_client = storage.Client()
         bucket = storage_client.get_bucket(output_bucket)
         blob_list = list(bucket.list_blobs(prefix=prefix))
-        logging.debug("Output files:")
         # should always be a single document here
         for i, blob in enumerate(blob_list):
             # If JSON file, download the contents of this blob as a bytes object.
             if blob.content_type == "application/json":
                 blob_as_bytes = blob.download_as_bytes()
 
                 document = documentai.types.Document.from_json(blob_as_bytes)
-                logging.debug(f"Fetched file {i + 1}")
+                logging.debug(f"Fetched file {i + 1}: {blob.name}")
             else:
                 logging.info(f"Skipping non-supported file type {blob.name}")
 
         # Delete the unique folder created for this operation
         blobs = list(bucket.list_blobs(prefix=prefix))
         bucket.delete_blobs(blobs)
-
         hitl_op_id = None
-        results_json = blob_as_bytes
+        if hitl_op_full_path:
+            logging.debug(f"Async processing returned hitl_op = {hitl_op_full_path}")
+            hitl_op_split = hitl_op_full_path.split('/')
+            hitl_op_id = hitl_op_split.pop()
```

Comment:
```suggestion
            hitl_op_id = hitl_op_full_path.split("/").pop()
```

Reply:
There's probably a function in the `Operation` library that parses it for you, but this is slightly cleaner